pkg/controller/plan/scheduler/vsphere/scheduler.go

```diff
@@ -161,10 +175,7 @@ func (r *Scheduler) buildInFlight() (err error) {
 			vm := &model.VM{}
 			err = r.Source.Inventory.Find(vm, vmStatus.Ref)
 			if err != nil {
-				if errors.As(err, &web.NotFoundError{}) {
-					continue
-				}
-				if errors.As(err, &web.RefNotUniqueError{}) {
+				if errors.As(err, &web.NotFoundError{}) || errors.As(err, &web.RefNotUniqueError{}) {
```

Comment:
nitpick: i like the two ifs better, for me it's easier to read two ifs then an if with or ...